packages/twenty-front/src/modules/activities/files/components/Attachments.tsx

```diff
@@ -57,6 +54,11 @@ const StyledFileInput = styled.input`
   display: none;
 `;
 
+const StyledDropZoneContainer = styled.div`
+  height: 100%;
+  padding: 24px;
```

Comment:
Same, we should use `theme.spacing`:

```suggestion
  padding: ${({ theme }) => theme.spacing(6)};
```